core/templates/dev/head/pages/exploration-player-page/layout-directives/progress-nav.directive.ts

```diff
@@ -103,6 +105,23 @@ angular.module('oppia').directive('progressNav', [
             }
           };
 
+          var doesInteractionHaveSpecialCaseForMobile = function() {
+            // Special cases to show submit button:
+            // 1. ItemSelectionInput interaction with maximum selectable
+            //    choices > 1 in desktop/mobile mode.
+            // 2. ItemSelectionInput interaction with maximum selectable
+            //    choices == 1 in mobile mode.
+            // 3. MultipleChoiceInput interaction in mobile mode.
```

Comment:
Could you structure this comment as you structure the ifs: 
```
The submit button should be show when:
  1. Mobile and ItemSelectionInput or MultipleChoiceInput
  2. Desktop and ItemSelectionInput with more than one answer
```
(these are not exact words just an outline)

Reply:
Done.